cloudconnectlib/splunktacollectorlib/data_collection/ta_config.py

```diff
@@ -46,7 +46,7 @@ def __init__(
         self._single_instance = single_instance
         self._task_configs = []
         self._client_schema = client_schema
-        self._server_info = sc.ServerInfo(
+        self._server_info = server_info.ServerInfo.from_server_uri(
```

Comment:
Can you please confirm if this change is expected apart from poetry updated?

Reply:
Yes, it should be here.
Let me clarify this in the description of the PR.